README.md

```diff
@@ -0,0 +1,124 @@
+# Cribl Search Grafana Plugin
+
+This is a Data Source Plugin for Grafana that pulls from Cribl Search.
+https://cribl.io/search/
+
+NOTE: The rest of this README was auto-generated via `npx @grafana/create-plugin@latest`.  This particular README provides
+some generalized guidance on _Grafana plugin development_.
+
+To learn more about the plugin from the an end user perspective, please refer to [src/README.md](src/README.md), which is
+the same info users see in the Grafana UI.
+
+All Cribl-specific source code and config lives in [src](src).
+
+## What are Grafana data source plugins?
+
+Grafana supports a wide range of data sources, including Prometheus, MySQL, and even Datadog. There’s a good chance you can already visualize metrics from the systems you have set up. In some cases, though, you already have an in-house metrics solution that you’d like to add to your Grafana dashboards. Grafana Data Source Plugins enables integrating such solutions with Grafana.
+
+## Getting started
+
+
+### Frontend
+
+1. Install dependencies
+
+   ```bash
+   npm run install
```

Comment:
Based on the comment above, this is autogenerated... however weirdly this command doesn't work for me, fwiw. I need to do `npm install` instead.

Reply:
Good catch, I'll update this